kolibri/content/api.py

```diff
@@ -57,7 +57,7 @@ def get_ancestor_topics(channel_id=None, content=None, **kwargs):
     :param content: ContentMetadata or str
     :return: QuerySet of ContentMetadata
     """
-    return content.get_ancestors().filter(kind="topic").using(channel_id)
+    return content.get_ancestors().filter(kind__kind="topic").using(channel_id)
```

Comment:
We should switch these types of things to constants. As an example of this:
Defining constants: https://github.com/learningequality/kolibri/blob/master/kolibri/auth/constants/collection_kinds.py
Using constants: https://github.com/learningequality/kolibri/blob/master/kolibri/auth/models.py#L72